Sources/FaunaDB/Field.swift

```diff
@@ -1,14 +1,50 @@
 internal let rootField = Field<Value>()
 
+/**
+    `Field` represents a field extractor for database entries returned by the
+    server.
+
+    For example:
+
+        let nameField = Field<String>("data", "name")
+
+        let user = try! client.query(
+            Get(
+                Ref("classes/user/42")
+            )
+        ).await(timeout: .now() + 5)
+
+        let userName = user.get(field: nameField)
+
+    Every field has a path which is composed by a sequence of segments.
+    A segment can be:
+
+    - `String`: when the desired field is contained into an object;
+    - `Int`: when the desired field is contained into an array.
```

Comment:
"into" => "in". Maybe also clarify that the desired field is the nth term of the array where n is the integer